crates/nomination/src/benchmarking.rs

```diff
@@ -41,58 +41,73 @@ benchmarks! {
         setup_exchange_rate::<T>();
         <NominationEnabled<T>>::set(true);
 
-        let vault: T::AccountId = account("Vault", 0, 0);
-        mint_collateral::<T>(&vault, (1u32 << 31).into());
-        assert_ok!(VaultRegistry::<T>::_register_vault(&vault, 100000000u32.into(), dummy_public_key(), DEFAULT_TESTING_CURRENCY));
+        let vault_id: VaultId<T::AccountId, _> = VaultId::new(
+            account("Vault", 0, 0),
+            T::GetGriefingCollateralCurrencyId::get(),
+            T::GetWrappedCurrencyId::get()
+        );
+        mint_collateral::<T>(&vault_id.account_id, (1u32 << 31).into());
+        assert_ok!(VaultRegistry::<T>::_register_vault(vault_id.clone(), 100000000u32.into(), dummy_public_key()));
 
-    }: _(RawOrigin::Signed(vault))
+    }: _(RawOrigin::Signed(vault_id.account_id), vault_id.currencies.collateral, vault_id.currencies.wrapped)
```

Comment:
Did you check if we could build our own origin for this? I asked Parity if it was possible but never heard back

Reply:
I think we could possibly change the Origin type, but it would change it for _all_ dispatchable functions, including those that have nothing to do with vault ids. We'd have to use an `enum{Vault(...), Account(...)` as the origin. I don't think the slight cleanup in the api is worth the additional complexity